_shared/project/requirements/dev.in

```diff
@@ -0,0 +1,12 @@
+pip-tools
+-r prod.txt
+factory-boy
+{% if cookiecutter.get("_directory") == "pyramid-app" %}
+pyramid-ipython
+{% else %}
+ipython
+{% endif %}
```

Comment:
It's good to have `ipython` installed in dev. It gives a more friendly and powerful shell when debugging with the standard Python `breakpoint()` builtin or when running `make shell`. For Pyramid apps we used `pyramid-ipython` as it adds some nice IPython-Pyramid integration. For non-web apps we don't want the Pyramid dependency so just install `ipython` directly.